paymentsheet/src/main/java/com/stripe/android/paymentsheet/state/PaymentSheetLoader.kt

```diff
@@ -393,13 +393,12 @@ private fun PaymentMethod.toPaymentSelection(): PaymentSelection.Saved {
     return PaymentSelection.Saved(this)
 }
 
-internal fun PaymentSheet.BillingDetailsCollectionConfiguration.toInternal():
-    BillingDetailsCollectionConfiguration {
+internal fun PaymentSheet.BillingDetailsCollectionConfiguration?.toInternal(): BillingDetailsCollectionConfiguration {
     return BillingDetailsCollectionConfiguration(
-        collectName = name == Always,
-        collectEmail = email == Always,
-        collectPhone = phone == Always,
-        address = when (address) {
+        collectName = this?.name == PaymentSheet.BillingDetailsCollectionConfiguration.CollectionMode.Always,
```

Comment:
We could also short circuit is `this` is null, and just use the default constructor for `BillingDetailsCollectionConfiguration` either way is fine.